src/linter/rules/invalid-change-version.mjs

```diff
@@ -14,6 +14,19 @@ const NODE_RELEASED_VERSIONS = env.NODE_RELEASED_VERSIONS?.split(',');
 const isValidReplaceMe = (version, length) =>
   length === 1 && version === 'REPLACEME';
 
+/**
+ * Checks if a given semantic version should be ignored.
+ * A version is considered ignored if its major version is 0 and minor version is less than 2.
+ *
+ * @param {string} version - The version to check.
+ * @returns {boolean|undefined} Returns true if the version is ignored, false otherwise.
+ */
+const isIgnoredVersion = version => {
+  if (version === 'v0.11.15') return true;
+  const { major, minor } = parse(version) || {};
```

Comment:
Note: We are intentionally not throwing on an invalid semver, as `NODE_RELEASED_VERSIONS` has the final say on what's a valid version (although, a non-semver valid version would never happen)